docs/src/content/addons-api-changelog.md

```diff
@@ -10,6 +10,27 @@ menu:
 
 We try to avoid them, but this page lists breaking changes in the mitmproxy addon API.
 
+## mitmproxy 9.0
+
+#### Logging
+
+We've deprecated mitmproxy's homegrown logging system in favor of Python's builtin `logging` module.
+This means that addons should now use standard logging functionality instead of `mitmproxy.ctx.log`:
+
+```python
+# Deprecated:
+from mitmproxy import ctx
+ctx.log.info("hello world")
+
+# New:
+import logging
+logging.info("hello world")
+```
+
+
+Accordingly, the `add_log` event has been removed. Developers who rely on log entries need to register their own 
```

Comment:
`We've deprecated ` vs. `has been removed`
Aren't they mutually exclusive? Or are we not following a deprecate-release-remove-release strategy here?

Reply:
Right now `ctx.log` is deprecated, but `add_log` has been removed. You're making a reasonable point though, I can add a bit of boilerplate to still trigger `add_log` for now and just emit a deprecation warning.